spec/features/notifications_spec.rb

```diff
@@ -17,7 +17,7 @@
     end
   end
 
-  context '自分のチェックインが「いいね！」されたとき' do
+  context '自分の記録が「いいね！」されたとき' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.